shared/templates/template_ANSIBLE_file_permissions

```diff
@@ -3,21 +3,12 @@
 # strategy = configure
 # complexity = low
 # disruption = low
-
-- name: Find {{{ PATH }}} file(s)
-  find:
-    paths: "{{ '{{{ PATH }}}' | dirname }}"
-    patterns: "{{ '{{{ PATH }}}' | basename }}"
-  register: files_found
-  tags:
-    @ANSIBLE_TAGS@
-
-- name: Set permissions
+- name: Ensure permission {{{ FILEMODE }}} on {{{ FILEPATH }}}
   file:
-    path: "{{ item.path }}"
-    mode: {{{ MODE }}}
+    path="{{item}}"
+    mode={{{ FILEMODE }}}
```

Comment:
Does this syntax work? I'm used to colons.
Also, there are no references to equals here: https://docs.ansible.com/ansible/2.5/modules/file_module.html#examples